.github/workflows/release.yml

```diff
@@ -9,20 +9,57 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+
+      # Create a temporary, uniquely named branch to push release info to
+      - name: create temporary branch
+        uses: peterjgrainger/action-create-branch@v2.2.0
+        id: create-branch
+        with:
+          branch: "release-from-${{ github.sha }}"
+          sha: "${{ github.sha }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # check out the repository afterwards
       - uses: actions/checkout@v3
 
+
+      # fetch branches and switch to the temporary branch
+      - name: debug
+        run: git fetch --all && git checkout --track origin/release-from-${{ github.sha }}
+
+
+      # create release info and push it upstream
       - name: conventional Changelog Action
         id: changelog
         uses: TriPSs/conventional-changelog-action@v3
         with:
-          github-token: ${{ secrets.CHANGELOG_RELEASE }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we will need to switch this back before merging, otherwise other Actions won't run

Reply:
Done.

Reply:
After reading up on the matter once more, I am sure we will need to use `GITHUB_TOKEN` for every step except the one creating the actual release within GitHub. That step will use `CHANGELOG_RELEASE` in order to trigger `deploy.yml`.

This is necessary in order to prevent endless CI circles from happening, as we create and merge PRs to main in the process, which would trigger another instance of the same job if we didn't use `GITHUB_TOKEN`.

See also [https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow).